app/src/test/java/com/globant/viewmodels/CharacterFragmentViewModelTest.kt

```diff
@@ -0,0 +1,135 @@
+package com.globant.viewmodels
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import com.globant.domain.entities.Comic
+import com.globant.domain.entities.MarvelCharacter
+import com.globant.domain.usecases.GetCharacterByIdUseCase
+import com.globant.domain.usecases.GetComicsUseCase
+import com.globant.domain.utils.Result
+import com.globant.utils.Data
+import com.globant.utils.Status
+import com.google.common.truth.Truth
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import kotlinx.coroutines.newSingleThreadContext
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.koin.test.AutoCloseKoinTest
+
+class CharacterFragmentViewModelTest : AutoCloseKoinTest() {
+
+    companion object {
```

Comment:
pls set the companion object at the bottom of the class

Reply:
done